### ISSUE TITLE
Target netstandard2.0

### ISSUE THREAD

**Issue by cocowalla** [kendallb/ExcelHelper#6]
There are warnings when using this library from a .NET Core or Standard project - would be good if this library could target `netstandard2.0`.

Technically .NET Standard 2.0 libraries are [compatible](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) with .NET Framework 4.6.1, but I have seen some issues in practise, do if you decide to go ahead with this, and still want .NET Framework 4.6.1 support, it might be a good idea to target both `net461` _and_ `netstandard2.0`.

**Comment by kendallb** [comment]
The C1 library cannot work in .Net standard (but they did just release a new .Net 5 version). I will see if I can re-target the non-C1 version for standard 2.0.

**Comment by kendallb** [comment]
Nope, not gonna work as we use System.Drawing as part of the API. The only path forward would be .net 5 at some point.
